package.json

```diff
@@ -105,6 +105,8 @@
     "@ethereumjs/tx": "^3.2.1",
     "@formatjs/intl-relativetimeformat": "^5.2.6",
     "@fortawesome/fontawesome-free": "^5.13.0",
+    "@keystonehq/bc-ur-registry-eth": "^0.6.8",
+    "@keystonehq/metamask-airgapped-keyring": "0.2.1",
```

Comment:
why the exact dep?

Reply:
```suggestion
    "@keystonehq/metamask-airgapped-keyring": "^0.2.1",
```

Reply:
because the keyring is our core dependence, we run all the testing based on that version. In this case we lock the version here. In the future if we update some code, only testing and verified by both team, the new version can be used.